README.md

```diff
@@ -1 +1,47 @@
-# restaurant
+## restaurant
+ This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
+
+
+ ## Screen shot of Page
+<img src="./src/asset/home.png">
+
+
+ ## Page live Demo Link
+
+ [link!](https://flamboyant-ptolemy-e38460.netlify.app/)
+
+ ## Built With
+
+ - Only Javascript 
+ - Webpack
+ - Used the DOM Manipulation.
+ - CSS
+ - Images
+ - Linters
+
+ ## How It work 
+  - First Open Your terminal and paste : git clone git@github com:ccobasi/restaurant.git.
+
+  - Thene cd to the Folder && Open it in your Texteditor.
```

Comment:
- Do you mean `Then`? Make this instruction clear enough.
- Correct spelling errors and properly format text. `Texteditor` is not a valid word and should also start with a small letter since it's not at the start of a sentence neither it is after a period.